.github/workflows/configs/test-integration-rosa-matrix.yaml

```diff
@@ -0,0 +1,29 @@
+matrix:
+    distro:
+      - name: OpenShift 4.15
+        type: openshift
+        # renovate: datasource=endoflife-date depName=red-hat-openshift versioning=regex:^4(\.(?<minor>\d+))?(\.(?<patch>\d+))?$
+        # /!\ Warning: When a new minor version of OpenShift is released, you must update all N-1, N-2, and N-3 versions in this matrix.
+        # rationale: Red Hat supports the last four minor versions of OpenShift. Therefore, to ensure compatibility and support, we must test against these versions. For more details, refer to the official support policy at https://endoflife.date/red-hat-openshift.          version: 4.15.13
+        version: 4.15.11
+        platform: rosa   
+
+      - name: OpenShift 4.14
+        type: openshift
+        # renovate: datasource=endoflife-date depName=red-hat-openshift versioning=regex:^4(\.(\d+))?(\.(?<patch>\d+))?$
```

Comment:
if you want to update `4.14`, I'm not sure it will work as `4.15` is a valid update based on the supplied regex.

Reply:
```
^4.14(\.(?<patch>\d+))?$
```